AdvLoggerPkg/Application/AdvancedLogDumper/LogDumperCommon.c

```diff
@@ -0,0 +1,341 @@
+/** @file
+LogDumperCommon.c
+
+This application will dump the AdvancedLog to a file.
+
+Copyright (C) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+#include <LogDumper.h>
+
+//
+// Parameters
+//
+STATIC CONST SHELL_PARAM_ITEM  ParamList[] = {
+  { L"-h", TypeFlag  },    // -h Help
+  { L"-r", TypeFlag  },    // -r Raw file
+  { L"-v", TypeFlag  },    // -v Verbose
+  { L"-o", TypeValue },    // -o output file
+  { NULL,  TypeMax   }
+};
+
+BOOLEAN                                           mFlagVerbose = FALSE;
+STATIC ADVANCED_LOGGER_ACCESS_MESSAGE_LINE_ENTRY  mAccessEntry = { 0 };
+extern EFI_HII_HANDLE                             gAdvLogHiiHandle;
+
+/**
+ * Requests the Advanced Logger variables and dumps the raw file to a file
+ * @param[in] FileHandle         The handle of the file we want to write to
+ * @param[in] Verbose            Whether debugging statements should be written out to console
+ * @retval EFI_SUCCESS           We were able to write to the file
+ * @retval EFI_OUT_OF_RESOURCES  We failed to allocate a buffer
+ * @retval EFI_NOT_FOUND         If AdvLogger didn't create a buffer or isn't installed
+ * @retval EFI_INVALID_PARAMETER The FileHandle was bad or null
+ * @retval Others                Errors passed from GetVariable or ShellWriteFile
+ */

```

Comment:
To match current conventions.

```suggestion
/**
  Requests the Advanced Logger variables and dumps the raw file to a file

  @param[in] FileHandle         The handle of the file we want to write to
  @param[in] Verbose            Whether debugging statements should be written out to console
  @retval EFI_SUCCESS           We were able to write to the file
  @retval EFI_OUT_OF_RESOURCES  We failed to allocate a buffer
  @retval EFI_NOT_FOUND         If AdvLogger didn't create a buffer or isn't installed
  @retval EFI_INVALID_PARAMETER The FileHandle was bad or null
  @retval Others                Errors passed from GetVariable or ShellWriteFile
 */

```